Form component for mask and truncate feild

indraraj opened the pull request:
- Added the FormFragmentComponent to handle the `column.truncate.to.(d+).chars` &  `column.mask.with.(d+).chars` fields.

- Using the value entered in `n:` field to update the key and value in `column:` to update the corresponding value of the key.

- If the value entered in only any one of the feild then the combined value is ignored.

mdrillin commented on ui/packages/ui/src/app/components/formHelpers/FormFragmentComponent.tsx:
```diff
@@ -0,0 +1,111 @@
+      <InputGroup>
+        <Grid>
+          <GridItem span={9}>
+            <Flex className={'form-fragment-component-checkbox'}>
```
Remove 'checkbox' from this className

mdrillin commented on ui/packages/ui/src/app/components/formHelpers/FormFragmentComponent.tsx:
```diff
@@ -0,0 +1,111 @@
+            <Flex>
+              <FlexItem>n:</FlexItem>
+              <FlexItem>
+                <TextInput
```
Since this needs to be a positive int, we could probably set "min={"1"}" on the TextInput.  Could possibly also use 'onKeyPress' as in FormDurationComponent, to prevent entry of '-' and '.'

mdrillin reviewed (approved):
I think this looks good, just a couple minor suggestions noted, plus 
- for the column list input - user can leave a trailing comma which stays with the value (maybe that should be trimmed off - not sure if backend is ok with it?)  
- I was trying to think of a more descriptive component name instead of FormFragmentComponent -  maybe FormMaskOrTruncateComponent ?

indraraj commented:
Incorporated the suggestions.

- I will confirm once with Rene regarding min={'1'} and trailing comma issue.

- For the component name, I was trying to use a general name, updated the name to `FormMaskOrTruncateComponent` as of now.
